sup3r/models/multi_step.py

```diff
@@ -432,7 +448,78 @@ def load(cls, spatial_model_dirs, temporal_model_dirs, verbose=True):
         return cls(s_models, t_models)
 
 
-class MultiStepSurfaceMetGan(SpatialThenTemporalGan):
+class SpatialGanThenLinearInterp(SpatialThenTemporalBase):
```

Comment:
SpatialThenTemporalGan loads models which takes model dirs as inputs to the load method. The linear model doesnt load this way. We could change it to load that way though.

Reply:
Yeah it makes a lot more sense to change LinearInterp.load() vs build a whole new multistep model. Load is designed to always load from disc anyway, the current LinearInterp.load() doesnt fit this paradigm (clearly it would be better architecture if it did!). This is a good example where we (me) violated polymorphism and it leads to more needlessly complicated architecture down the road.

Reply:
Lol yeah I thought about changing the linear load method but didnt want to mess with the models youve already built. Ill change it though.

Reply:
oh go for it haha this is a great example where i did something without thinking too hard about it. Also no time lost since i didnt have to train a linear model or anything.